src/openrct2/ride/RideData.h

```diff
@@ -199,12 +199,30 @@ struct RideOperatingSettings
 {
     uint8_t MinValue;
     uint8_t MaxValue;
-    uint8_t MaxBrakesSpeed;
+    uint8_t OperatingSettingMultiplier = 1; // Used for the Ride window, cosmetic only.
+};
+
+struct RideTrackSpeedSettings
+{
+    int8_t BrakesMaxSpeed = 30;
+    int8_t BoosterMaxSpeed = 30;
+};
+
+struct RideBoosterSettings
+{
     uint8_t PoweredLiftAcceleration;
```

Comment:
Also zero-initialise these two.

Reply:
I have zero-initialized these which are intended to be changed as part of https://github.com/OpenRCT2/OpenRCT2/discussions/21752